### DISCUSSION TITLE
Image corrupt?

### DISCUSSION THREAD

**Discussion by Del6747432** [bareboat-necessities/lysmarine_gen#202] [General]
Downloaded the image in parts,
Extracted the parts to one .IMG (6 parts in total )
Used rasp pi image tool.

On first boot after a minute, all I get is authentication failed. 

Tried several attempts, any suggestions.

Thanks

**Comment by mgrouch** [comment]
How did you burn it? You do not need unarchive it from .xz. Rpi-imager can handle .xz. Do not use any new options of rpi imager to set user or password. Just burn it as is. 
you can check md5sum on .xz file and compare it with md5sum of .xz published on cloudsmith. It must match.

**Reply by Del6747432** [reply]
Thanks for swift reply.

I just used raspberry pi image to burn it.

I'll try again, on raspberry pi image maker, but I'll choose the first part of the 6 parts I downloaded?

**Reply by mgrouch** [reply]
> Thanks for swift reply.
> 
> I just used raspberry pi image to burn it.
> 
> I'll try again, on raspberry pi image maker, but I'll choose the first part of the 6 parts I downloaded?

Why would you burn just part? you need to burn whole .xz file which is about 2.8 Gb

**Reply by Del6747432** [reply]
Ok, on Raspberry image maker, you choose custom image, then direct it to the file you want to make.
The image is downloaded in 6 parts. 

Which file do I point the image maker to?

**Reply by mgrouch** [reply]
There is a script how to merge them in release notes. Or alternative download location from archive.org of the whole thing. 

Did you read it?

https://github.com/bareboat-necessities/lysmarine_gen/releases/tag/v2022-07-13

you need to open .xz file in rpi imager

**Reply by Del6747432** [reply]
Downloaded the complete from archive.org. working first time. Thanks for you help and thanks for this software. Appreciated.
